packages/fxa-settings/src/pages/Signin/SigninUnblock/index.tsx

```diff
@@ -49,10 +54,19 @@ export const SigninUnblock = ({
     ResendStatus.none
   );
 
+  const alertBar = useAlertBar();
   const ftlMsgResolver = useFtlMsgResolver();
   const location = useLocation();
   const navigate = useNavigate();
 
+  const webRedirectCheck = useWebRedirect(integration.data.redirectTo);
```

Comment:
Do we need to check if `redirectTo` exist first?  That `data` is an `any` so the type system isn't gonna save us here.

Reply:
Right, I checked inside `useWebRedirect` because hooks can't be called conditionally. Maybe the better solution would have been to make it a function instead of a hook 🤔

Reply:
It had to be a hook unless we wanted to pass `config`, `location`, and `ftlMsgResolver` into it every time.

Reply:
Ah yeah, that makes sense.